test/include/nntrainer_test_util.h

```diff
@@ -191,6 +191,20 @@ class ScopedIni {
     }                                             \
   } while (0)
 
+#define GEN_TEST_INPUT_DOUBLE(input, equation_i_j_k_l)        \
```

Comment:
Please take batch, channel, height, width as arguments. This damages readability and has potential errors for other developers later. 

This doesn't incur runtime overhead.